tools/packaging/common/envoy_bootstrap.json

```diff
@@ -550,6 +550,10 @@
             "port_value": {{ .envoy_prometheus_port }}
           }
         },
+        {{- if .bypass_overload_manager}}
+        "ignore_global_conn_limit": true,
+        "bypass_overload_manager": true,
+        {{ end}}
```

Comment:
And you did not set `overload_manager`

Reply:
I think this is to allow bypassing overload manager if/when it is configured. In this PR I am not configuring it. If folks use overload manager via https://github.com/istio/istio/issues/14366#issuecomment-548141596 - they get this benefit.